packages/gatsby/package.json

```diff
@@ -0,0 +1,33 @@
+{
+  "name": "@vercel/gatsby",
+  "version": "0.1.0",
+  "license": "MIT",
+  "main": "dist/index.js",
+  "homepage": "https://vercel.com/docs",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/vercel/vercel.git",
+    "directory": "packages/gatsby"
+  },
+  "scripts": {
+    "build": "node build",
+    "test": "jest --env node --verbose --bail --runInBand",
+    "test-integration-once": "yarn test",
+    "prepublishOnly": "node build"
+  },
+  "files": [
+    "dist"
+  ],
```

Comment:
We could also add this to make sure it publishes public (the first publish is always private)

```
"publishConfig": {
    "access": "public"
  }
```